components/enums/EnumSocial.ts

```diff
@@ -0,0 +1,12 @@
+enum EnumSocial {
+  WEBSITE = 'Website',
+  GITHUB = 'GitHub',
+  TWITTER = 'Twitter',
+  LINKEDIN = 'LinkedIn',
+  REDDIT = 'reddit',
```

Comment:
chore (blocking): Consistency is key. Write 'Reddit' instead of 'reddit'

Reply:
I used to write reddit in lowercase because they often use these notation.
Further investigation showed that you can also use the uppercase notation. I have changed the literal to be consistent with the other socials 👍